src/content/docs/en/basics/astro-pages.mdx

```diff
@@ -73,9 +73,11 @@ import MySiteLayout from '../layouts/MySiteLayout.astro';
 
 Astro also treats any Markdown (`.md`) files inside of `src/pages/` as pages in your final website. If you have the [MDX Integration installed](/en/guides/integrations-guide/mdx/#installation), it also treats MDX (`.mdx`) files the same way. These are commonly used for text-heavy pages like blog posts and documentation.
```

Comment:
> These are commonly used for text-heavy pages like blog posts and documentation.

I wonder if this sentence contradicts the tip given below since we recommend content collections for blog posts.
But I don't know what is the best use of Markdown pages... Maybe "standalone pages" (e.g. `/about`, `/legal-notice`)?